src/main/java/ch/njol/skript/registrations/EventValues.java

```diff
@@ -454,12 +457,31 @@ public static boolean doesEventValueHaveTimeStates(Class<? extends Event> event,
 		return getEventValueConverter(event, c, TIME_PAST, false) != null || getEventValueConverter(event, c, TIME_FUTURE, false) != null;
 	}
 
-	private record EventValueInfo<E extends Event, T>(
+	/**
+	 * @return All the event values for each registered event's class.
+	 */
+	public static Multimap<Class<? extends Event>, EventValueInfo<?, ?>> getPerEventEventValues() {
+		Multimap<Class<? extends Event>, EventValueInfo<?, ?>> eventValues = MultimapBuilder
+			.hashKeys()
+			.hashSetValues()
+			.build();
+
+		for (int i = TIME_PAST; i <= TIME_FUTURE; i++) {
```

Comment:
Can EventValues return the possible time integers in the case that another event value is ever added. Something like EventValues#getTimeStates